Tests/UIKitTests/UIColorExtensionsTests.swift

```diff
@@ -124,6 +124,32 @@ class UIColorExtensionsTests: XCTestCase {
 
     }
 
+    func testCoreImageColor() {
```

Comment:
We're not running watchOS tests currently, but ideally, of course this should be wrapped, @SD10 thanks for bringing this up, didn't notice 😬.
@tache Could you please wrap the test function as well 😅